<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>intelligent Video Sampler 3D</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/icons/ivs3dIcon</normaloff>:/icons/ivs3dIcon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionSave_Project"/>
    <addaction name="actionSave_Project_As"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_Input"/>
    <addaction name="actionOpen_Input_Video"/>
    <addaction name="actionOpen_Meta_Data"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSet_Input_Path"/>
    <addaction name="separator"/>
    <addaction name="actionToggleTheme"/>
    <addaction name="actionUse_CUDA"/>
    <addaction name="actionCreate_log_file"/>
    <addaction name="actionInterpolate_missing_meta_data"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionHelp"/>
    <addaction name="actionLicence"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionReset_Boundaries"/>
    <addaction name="separator"/>
    <addaction name="actionDelete_Keyframes"/>
    <addaction name="actionDelete_All_Keyframes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionSave_Project_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save Project As ...</string>
   </property>
  </action>
  <action name="actionOpen_Input">
   <property name="text">
    <string>Open Input Folder</string>
   </property>
  </action>
  <action name="actionSet_Reconstructiontool_Path">
   <property name="text">
    <string>Manage reconstruction tools </string>
   </property>
  </action>
  <action name="actionSet_Input_Path">
   <property name="text">
    <string>Set input Path</string>
   </property>
  </action>
  <action name="actionChange_layout_style">
   <property name="text">
    <string>Change layout style</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionOpen_Input_Video">
   <property name="text">
    <string>Open Input Video</string>
   </property>
  </action>
  <action name="actionToggleTheme">
   <property name="text">
    <string>Toggle theme</string>
   </property>
   <property name="toolTip">
    <string>Toggle color theme between dark and light mode</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionUse_CUDA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use CUDA</string>
   </property>
  </action>
  <action name="actionLicence">
   <property name="text">
    <string>Licence</string>
   </property>
  </action>
  <action name="actionStartAutoExec">
   <property name="text">
    <string>Start automatic execution</string>
   </property>
   <property name="toolTip">
    <string>Start automatic execution</string>
   </property>
  </action>
  <action name="actionSetupAutoExec">
   <property name="text">
    <string>Setup automatic execution</string>
   </property>
  </action>
  <action name="actionCreate_log_file">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create log file</string>
   </property>
  </action>
  <action name="actionOpen_Meta_Data">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open Meta Data</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionReset_Boundaries">
   <property name="text">
    <string>Reset Boundaries</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDelete_All_Keyframes">
   <property name="text">
    <string>Reset Keyframes</string>
   </property>
   <property name="toolTip">
    <string>Resets keyframes to default</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionDelete_Keyframes">
   <property name="text">
    <string>Delete Keyframes inside boundaries</string>
   </property>
   <property name="toolTip">
    <string>Delete keyframes within boundaries</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Del</string>
   </property>
  </action>
  <action name="actionInterpolate_missing_meta_data">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Interpolate missing meta data</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
